src/rules/__tests__/no-untyped-mock-factory.test.ts

```diff
@@ -0,0 +1,137 @@
+import path from 'path';
+import { ESLintUtils } from '@typescript-eslint/utils';
+import dedent from 'dedent';
+import rule from '../no-untyped-mock-factory';
+
+function getFixturesRootDir(): string {
+  return path.join(__dirname, 'fixtures');
+}
+
+const rootPath = getFixturesRootDir();
+
+const ruleTester = new ESLintUtils.RuleTester({
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    sourceType: 'module',
+    tsconfigRootDir: rootPath,
+    project: './tsconfig.json',
+  },
+});
```

Comment:
We shouldn't be configuring a TypeScript project since that's not needed by this rule (+ let's switch to using `TSESLint` because that's what we use in our other tests):

```suggestion
import { TSESLint } from '@typescript-eslint/utils';
import dedent from 'dedent';
import rule from '../no-untyped-mock-factory';

const ruleTester = new TSESLint.RuleTester({
  parser: require.resolve('@typescript-eslint/parser'),
});
```